hack/preload-images/generate-preloaded-images-tar.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex 
+
+PROFILE=generate-preloaded-images-tar
+KUBERNETES_VERSION=${KUBERNETES_VERSION:-""}
+TARBALL_FILENAME=preloaded-images-k8s-$KUBERNETES_VERSION.tar
+
+function delete_minikube {
+    out/minikube delete --profile=$PROFILE
+}
+
+trap "delete_minikube" ERR
+
+out/minikube start --memory=10000 --profile=$PROFILE --kubernetes-version=$KUBERNETES_VERSION
```

Comment:
is that 10gb ram ?

Reply:
yup, it's needed since 1.6 G of files needs to be tarred up within the VM